src/scene.lisp

```diff
@@ -100,11 +106,13 @@
                (setf (gethash (car object) (objects ,sym)) (cadr object)))
              ,sym)))))
 
-(defmacro make-scene (assets objects)
+(defmacro make-scene ((&key (active t) (free :now) (gpu t)) assets objects)
```

Comment:
I think `(active t)` and `(free :now)` are obvious defaults, but a little less sure of `(gpu t)`. I think it causes less problems for a new user and it doesn't really cost anything to defer loading until `do-game-loop` starts, but I just wanted to mention it.

Reply:
Is there a reason you chose to add the keywords in a list at the beginning of the arguments (which is a breaking API change), as opposed to tacking them onto the end (which would not be)? Not that anyone is using this right now to actually break anything. But we'd still have to fix the examples and these are things we need to consider going forward.

Also, I know I'm the one who suggested `gpu`, but I think something like `delay-init` would be clearer. (Or `gpu-context-required` but that's just too long.)

Also also, I agree with your defaults, including the last one.

Reply:
> Is there a reason you chose to add the keywords in a list at the beginning of the arguments

Yes, but like always it could be a bad reason: it looked weird like they were just floating out in space when I put them after big lists of assets & objects :sweat_smile: But I agree that this is a little silly just to have them visually at the top of a make-scene, I'll change that.

> I think something like `delay-init` would be clearer.

Agreed, I considered it but stuck with `gpu`. I'll fix that.

> I agree with your defaults

Okay, great!